x-pack/plugins/infra/server/lib/alerting/metric_threshold/lib/metric_query.ts

```diff
@@ -84,12 +85,11 @@ export const getElasticsearchMetricQuery = (
   afterKey?: Record<string, string>,
   fieldsExisted?: Record<string, boolean> | null
 ) => {
-  const { metric, aggType } = metricParams;
-  if (aggType === Aggregators.COUNT && metric) {
```

Comment:
The count aggregator does not have a metric in its type, so I removed this condition. (Typescript was giving me an error)